images/miq-app-frontend/docker-assets/manageiq-http.conf

```diff
@@ -0,0 +1,29 @@
+## ManageIQ HTTP Virtual Host Context
+
+# Timeout: The number of seconds before receives and sends time out.
+Timeout 120
+
+# HTTP Start-up error log
+ErrorLog /var/www/miq/vmdb/log/apache/miq_apache.log
+
+# Disable this section if using HTTP only
+RewriteEngine On
+Options SymLinksIfOwnerMatch
+
+<VirtualHost *:80>
+  DocumentRoot /var/www/miq/vmdb/public
+  Include conf.d/manageiq-redirects-ui
+  Include conf.d/manageiq-redirects-ws
+  Include conf.d/manageiq-redirects-websocket
```

Comment:
I think we will probably still need cockpit here right?

I'm not sure if they are all tested in containers, but I think that's what https://github.com/ManageIQ/manageiq-pods/issues/97 is meant to address, so maybe we can investigate in a followup?